patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -16,17 +16,7 @@
      public static final AxisAlignedBB field_185505_j = new AxisAlignedBB(0.0D, 0.0D, 0.0D, 1.0D, 1.0D, 1.0D);
      @Nullable
      public static final AxisAlignedBB field_185506_k = null;
-@@ -91,7 +92,8 @@
- 
-     public static Block func_149729_e(int p_149729_0_)
-     {
--        return (Block)field_149771_c.func_148754_a(p_149729_0_);
-+        Block ret = (Block)field_149771_c.func_148754_a(p_149729_0_);
-+        return ret == null ? net.minecraft.init.Blocks.field_150350_a : ret;
```

Comment:
The block registry already returns `Blocks.field_150350_a` as a missing default, never `null`, so this was redundant.